src/app/screens/Login/index.tsx

```diff
@@ -1,5 +1,84 @@
 import React from 'react';
+import i18next from 'i18next';
+import { useForm } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+
+import { User } from '~utils/types';
+import { login } from '~services/UserService';
+import { useLazyRequest } from '~app/hooks/useRequest';
+import Loading from '~components/Spinner/components/loading';
+import CustomErrorDisplayer from '~components/CustomErrorDisplayer';
+import { PATHS } from '~constants/paths';
+
+import CustomInput from '../../components/CustomInput';
+import WoloxImg from '../Assets/wolox-logo.png';
+
+import { LOGIN_FIELDS } from './constants';
+import styles from './styles.module.scss';
 
 export default function Login() {
-  return <span>Login, ¡coming soon!</span>;
+  const { register, handleSubmit, errors } = useForm<User>({ mode: 'all' });
+  const [state, loading, error, sendRequest] = useLazyRequest({
+    request: login
+  });
+  // eslint-disable-next-line no-console
+  console.log(state?.headers);
+  const disableButton = () => {
```

Comment:
this should be a value variable not a function, remember that on every re render this will separate a new memory reference for this.